src/app/service/payment.service.ts

```diff
@@ -1,21 +1,39 @@
-import { Injectable } from '@angular/core';import { Observable } from 'rxjs/Observable';
-import { DatastoreService } from './datastore.service';
-import { Payment } from '../models/payment';
+import {Injectable} from '@angular/core';
+import {DatastoreService} from './datastore.service';
+import {Payment} from '../models/payment';
+import {Http} from '@angular/http';
```

Comment:
The spacing here is wrong. It should be `{ Injectable }`
That goes for the other ones as well :)

Reply:
Fixing...